src/components/App.styl

```diff
@@ -0,0 +1,5 @@
+@require 'settings/breakpoints'
+
+.AppSidebar
+    +small-screen()
+        z-index 80 !important
```

Comment:
Mmh it is maybe better to do this at modal level since here every fullscreen modal will not be completely fullscreen and this may be a behavior we want. Let's add a comment here saying why we have done that and open an issue on cozy-ui :)

Reply:
:+1: :heavy_check_mark: See the issue here : https://github.com/cozy/cozy-ui/issues/482